health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/SetupCampaign.js

```
@@ -692,47 +947,67 @@ const SetupCampaign = ({ hierarchyType, hierarchyData }) => {
-          data: draftData,
-        });
+        history.push(
+          `/${window.contextPath}/employee/campaign/update-campaign?key=1&parentId=${draftData?.id}&campaignName=${draftData?.campaignName}`,
+          {
+            name: draftData?.campaignName,
+            projectId: draftData?.projectId,
+            data: draftData,
+          }
+        );
+        break;
+      case "HCM_CONFIGURE_APP":
```

Biome (1.9.4)

[error] Duplicate case label.

The first similar label is here:


(lint/suspicious/noDuplicateCase)